src/Identity/EntityFrameworkCore/test/EF.Test/MaxKeyLengthSchemaTest.cs

```diff
@@ -24,59 +23,73 @@ public MaxKeyLengthSchemaTest(ScratchDatabaseFixture fixture)
 
             services
                 .AddSingleton<IConfiguration>(new ConfigurationBuilder().Build())
-                .AddDbContext<IdentityDbContext>(o =>
+                .AddDbContext<VerstappenDbContext>(o =>
                     o.UseSqlite(fixture.Connection)
                         .ConfigureWarnings(b => b.Log(CoreEventId.ManyServiceProvidersCreatedWarning)))
                 .AddIdentity<IdentityUser, IdentityRole>(o => o.Stores.MaxLengthForKeys = 128)
-                .AddEntityFrameworkStores<IdentityDbContext>();
+                .AddEntityFrameworkStores<VerstappenDbContext>();
 
             services.AddLogging();
 
-            var provider = services.BuildServiceProvider();
-            _builder = new ApplicationBuilder(provider);
+            _builder = new ApplicationBuilder(services.BuildServiceProvider());
 
-            using (var scoped = provider.GetRequiredService<IServiceScopeFactory>().CreateScope())
-            using (var db = scoped.ServiceProvider.GetRequiredService<IdentityDbContext>())
+            using (var scope = _builder.ApplicationServices.GetRequiredService<IServiceScopeFactory>().CreateScope())
             {
-                db.Database.EnsureCreated();
+                scope.ServiceProvider.GetRequiredService<VerstappenDbContext>().Database.EnsureCreated();
             }
         }
 
-        [ConditionalFact(Skip = "https://github.com/aspnet/AspNetCore/issues/7925")]
+        // Need a different context type here since EF model is changing by having MaxLengthForKeys
+        public class VerstappenDbContext : IdentityDbContext<IdentityUser, IdentityRole, string>
+        {
```

Comment:
Random question, what is a `Verstappen`?

Reply:
It's just a different way of saying `Max`. And this is a context for testing MaxKeyLength.

Reply:
A Formula1 inside joke. Nice 😃

Reply:
🙄